lib/src/oauth2_flows/auth_code.dart

```diff
@@ -15,20 +15,23 @@ import '../http_client_base.dart';
 import '../typedefs.dart';
 import '../utils.dart';
 
-// The OAuth2 Token endpoint can be used to make requests as
-//    https://www.googleapis.com/oauth2/v2/tokeninfo?access_token=<token>
-//
-// A successfull response from the server will give an HTTP response status
-// 200 and a body of the following type:
-// {
-//   "issued_to": "XYZ.apps.googleusercontent.com",
-//   "audience": "XYZ.apps.googleusercontent.com",
-//   "scope": "https://www.googleapis.com/auth/bigquery",
-//   "expires_in": 3547,
-//   "access_type": "offline"
-// }
-//
-// Scopes are separated by spaces.
+/// The OAuth2 Token endpoint can be used to make requests as
+///    https://www.googleapis.com/oauth2/v2/tokeninfo?access_token=<token>
+///
+/// A successful response from the server will give an HTTP response status
+/// 200 and a body of the following type:
+///
+/// ```json
+/// {
+///   "issued_to": "XYZ.apps.googleusercontent.com",
+///   "audience": "XYZ.apps.googleusercontent.com",
+///   "scope": "https://www.googleapis.com/auth/bigquery",
+///   "expires_in": 3547,
+///   "access_type": "offline"
+/// }
+/// ```
+///
+/// Scopes are separated by spaces.
```

Comment:
Please revert this change. It's **not** a dartdoc. It's intended to understand the body of the method.

Reply:
ack